packages/db/lib/gen-migration.mjs

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import { stat } from 'fs/promises'
+import path from 'path'
+import loadConfig from './load-config.mjs'
+
+async function generateMigration (_args) {
+  const { configManager } = await loadConfig({}, _args)
+  const migrationsDir = configManager.current.migrations.dir
+
+  // Check migrations directory exists
+  await checkMigrationsDirectoryExists(migrationsDir)
+
+  try {
+    await configManager.parseAndValidate()
+    const files = await fs.promises.readdir(migrationsDir)
+
+    const nextMigrationId = getNextMigrationId(files)
+
+    const nextDoMigrationName = `${nextMigrationId.toString().padStart(3, '0')}.do.sql`
+    await generateMigrationFile(migrationsDir, nextDoMigrationName)
+
+    const nextUndoMigrationName = `${nextMigrationId.toString().padStart(3, '0')}.undo.sql`
```

Comment:
@mcollina Should we generate also undo migration?

Reply:
Yes